<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>Enter the password</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="passwordBox">
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="loginButton">
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>All Tables:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="numOfTablesLabel">
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listOfTables">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;Tahoma&quot;;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="deleteTableButton">
                <property name="text">
                 <string>Delete Table</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="addTableButton">
                <property name="text">
                 <string>Add Table</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>All Questions: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="numOfQuestionsLabel">
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="settingsButton">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listOfQuestions">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;Tahoma&quot;;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addQuestionButton">
              <property name="text">
               <string>Add Question</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="deleteQuestionButton">
                <property name="text">
                 <string>Delete Question</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="editQuestionButton">
                <property name="text">
                 <string>Edit Question</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <spacer name="verticalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_11">
                    <item>
                     <spacer name="horizontalSpacer_5">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="resetButton">
                      <property name="text">
                       <string>Reset</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <item>
                       <widget class="QLabel" name="label_4">
                        <property name="styleSheet">
                         <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
                        </property>
                        <property name="text">
                         <string>Name of Question</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="nameOfQuestionEdit">
                        <property name="styleSheet">
                         <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_7">
                      <item>
                       <widget class="QRadioButton" name="firstAnswerCorrect">
                        <property name="text">
                         <string/>
                        </property>
                        <property name="checked">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="firstAnswer">
                        <property name="styleSheet">
                         <string notr="true">font: 12pt &quot;Tahoma&quot;;</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_10">
                      <item>
                       <widget class="QRadioButton" name="secondAnswerCorrect">
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="secondAnswer">
                        <property name="styleSheet">
                         <string notr="true">font: 12pt &quot;Tahoma&quot;;</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_9">
                      <item>
                       <widget class="QRadioButton" name="thirdAnswerCorrect">
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="thirdAnswer">
                        <property name="styleSheet">
                         <string notr="true">font: 12pt &quot;Tahoma&quot;;</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_8">
                      <item>
                       <widget class="QRadioButton" name="fourthAnswerCorrect">
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="fourthAnswer">
                        <property name="styleSheet">
                         <string notr="true">font: 12pt &quot;Tahoma&quot;;</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QPushButton" name="backButton">
                    <property name="text">
                     <string>Back</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="saveQuestionButton">
                    <property name="text">
                     <string>Save</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
